src/components/NewLocationPage/NotesBlock/NotesBlock.tsx

```diff
@@ -6,22 +6,45 @@ import {
   IconWrapper,
   TextContainer,
   TextComponent,
+  StyledBaseButton,
 } from './NotesBlock.style';
+import { EventAction, EventCategory } from 'components/Analytics';
 
-const NotesBlock: React.FC<{ icon: any; title: string }> = ({
+interface NotesProps {
+  icon: any;
```

Comment:
Can we find the correct type? 😁

Reply:
`JSX.Element` perhaps?

Reply:
haha typescript is making me unhappy with this one. adding a todo and will be sure to investigate before 🚀

Reply:
oh whoops--didnt refresh and see your 2nd comment before replying. i think `JSX.Element` does the trick 💫🙏🏼

Reply:
👍  

FWIW- To find the type, I just added:

`const foo = (<VulnerabilityIcon />);`

and then hovered over `foo` in VSCode to see what type it said it was.  Not sure if this always works 100% (I suspect it may give you a more complicated type than necessary sometimes), but it can be helpful.

Reply:
oh sweet